src/main/scala/edu/berkeley/ce/sparkrocks/Block.scala

```diff
@@ -138,6 +138,116 @@ object Block {
       DenseMatrix.eye[Double](3)
     }
   }
+
+  /**
+    * Compares pointA to pointB. If pointA is first relative to pointB rotating counter-clockwise about center,
+    * the method returns true. Input vectors should have same z-coordinate - comparison is based in 2-D
+    *
+    * @param pointA Point of interest
+    * @param pointB Point of comparison
+    * @param center Reference point for comparison
+    * @return Returns TRUE if pointA is first relative to pointB, FALSE otherwise
+    */
+  private def ccwCompare(pointA: Array[Double], pointB: Array[Double],
+                         center: Array[Double]): Boolean = {
+    // Check that points are in the same x-y plane
+    if (math.abs(NumericUtils.roundToTolerance(pointA(2), 5) -
```

Comment:
Is there a reason we can't use the default tolerance of 6 in `roundToTolerance`?